.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,43 @@
 <!-- Please review the following before submitting a PR:
 Che's Contributing Guide: https://github.com/eclipse/che/blob/master/CONTRIBUTING.md
 Pull Request Policy: https://github.com/eclipse/che/wiki/Development-Workflow#pull-requests
-
-COMMITTERS: please include labels on each PR. Labels are listed here: https://github.com/eclipse/che/wiki/Labels but at a minimum you should include `kind/..` and `Dev Open Pull Request Status` labels.
 -->
 
 ### What does this PR do?
 
 
+### Screenshot/screencast of this PR
+<!-- Please include a screenshot or a screencast explaining what is doing this PR -->
+
+
 ### What issues does this PR fix or reference?
+<!-- Please include any related issue from eclipse che repository (or from another issue tracker).
+     Include link to other pull requests like documentation PR from [the docs repo](https://github.com/eclipse/che-docs)
+-->
+
+
+### How to test this PR?
+<!-- Please explain for example :
+  - The test platform (openshift, kubernetes, minikube, CodeReady Container, docker-desktop, etc)
+  - Installation method: chectl / che-operator
+  - steps to reproduce
+ -->
+
 
-<!-- #### Changelog -->
-<!-- The changelog will be pulled from the PR's title. 
-     Please provide a clear and meaningful title to the PR and don't include issue number -->
+### PR Checklist
 
+As the author of this Pull Request I made sure that:
 
-#### Release Notes
-<!-- markdown to be included in marketing announcement - N/A for bugs -->
+- [ ] The Eclipse Contributor Agreement is valid
+- [ ] Code produced is complete
+- [ ] Code builds without errors
+- [ ] Add new automated test for every bug fixed
+- [ ] The repository devfile is up to date and works
+- [ ] Sections `What issues does this PR fix or reference` and `How to test this PR` completed
+- [ ] Relevant user documentation updated
+- [ ] Relevant contributing documentation updated
+- [ ] CI/CD changes implemented, documented and communicated
```

Comment:
Do we really need checkboxes here? GitHub shows a progress bar if description has them. So, the progress bar will be always shown on each PR which looks weird for me... Maybe we should go with just a list?

Reply:
I think the idea was to really emphasis that items need to be done before merging it

so it may be expected to see this if all items are done
![image](https://user-images.githubusercontent.com/436777/90408628-5e555680-e0a8-11ea-8ba5-96c511898d89.png)

and this if you didn't complete all the items:
![image](https://user-images.githubusercontent.com/436777/90408834-b2603b00-e0a8-11ea-9a93-3ce8c5f69dd8.png)